### PR TITLE
APR-2548_code complate

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

**Review by martin-lush — APPROVED** [review]

File: src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs [hunk 1]
```diff
@@ -64,6 +63,9 @@ public async Task<IActionResult> ProcessApplicationStatus(Guid applicationId)
                 case ApplicationStatus.Withdrawn:
                     return View("~/Views/Roatp/ApplicationWithdrawn.cshtml", model);
                 case ApplicationStatus.Removed:
+                    var oversightStatus = await _overallOutcomeService.GetOversightReviewStatus(applicationId);
```
Comment (chrisfoster76) [inline src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs:66]: You've added a new method `GetOversightReviewStatus` and used it here, but why not just use `GetOversightReview` instead? That would save adding new methods, and the performance difference would be negligible.

Also it seems odd to get the review using the api client, but the status via the overall outcome service, no?

File: src/SFA.DAS.ApplyService.Web/Services/OverallOutcomeService.cs [hunk 2]
```diff
@@ -123,7 +126,11 @@ public async Task<ApplicationSummaryWithModeratorDetailsViewModel> BuildApplicat
             return model;
         }
 
-
+        public async Task<OversightReviewStatus?> GetOversightReviewStatus(Guid applicationId)
```
Comment (chrisfoster76) [inline src/SFA.DAS.ApplyService.Web/Services/OverallOutcomeService.cs:129]: Why would this be nullable? Isn't that what the `None` entry is for?

File: src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs:66]: yeah, I'm mixing and matching.  Will conform

File: src/SFA.DAS.ApplyService.Web/Services/OverallOutcomeService.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.ApplyService.Web/Services/OverallOutcomeService.cs:129]: taking this out

**Review by martin-lush — DISMISSED** [review]

**Review by martin-lush — APPROVED** [review]

**Pull request merged**